docs/getting-started/designing-integrations.md

```diff
@@ -3,12 +3,147 @@ title: Designing Integrations
+get started on their data automation journeys. In this section we have included
+the process/workflow we follow when embarking on the design phase of a new
+project.
 
-Since 2014 OpenFn's Global Services Team has helped hundreds of organizations get started on their data automation joruneys. 
+Let's start with a question:
 
-In this section we have included the process/workflow we follow when embarking on a new project. 
+### What is "data integration"?
 
+The term "data integration" is thrown around a lot and many people are
+intimidated by it. Don't be. Put simply, data integration means bringing
+together different bits of information.
 
+When thinking about data integration, you might envision a developer sitting
+behind a computer screen, writing code to move huge data sets between two
+databases. While this is _part_ of the story, it isn't the whole story. Data
+integration often involves a technical component, but there's an awful lot of
+non-technical design, planning, and analysis work that must be put in to combine
+data from different sources and create meaningful and valuable information.
 
+:::tip
 
+Data integration is not just the technical process of connecting bits of
+information; it's also the business and human processes that are involved in
+transferring _meaning_ from one user (or group of users) to another. Far more
+important than having a `patient_id` show up in System B is have the people who
+use System A and System B come to agreement on what a `patient` is in the first
+place and why anyone should be sharing their `id`!
+
+An integration is **only** as valuable as the processes or analyses it enables.
+
+:::
+
+### Multiple technologies to get the job done
+
+You are probably viewing this page because your organization is using multiple
+technologies to get your work done.
+
+![Designing_Integrations](/img/designing_integration1.png)
+
+<!-- source for diagram: https://docs.google.com/presentation/d/1mVwxE7-0-B5NzTII3nCFaTw2DimGDcFlUQhIbwDE2TU/edit?usp=sharing -->
+
+Perhaps you are using Open Data Kit (ODK) or SurveyCTO for mobile data
+collection and Salesforce or dhis2 as a management-information-system (MIS).
+Perhaps you are taking advantage of the powerful analytics and visualization
+capabilities of Metabase, Tableau or PowerBI. You might even be using an SMS
+service like telerivet or nexmo to stay connected with patients, customers, or
+employees.
+
+While there are lots of different tools that serve valuable purposes for their
+various user groups, those purpose are often related and we find ourselves in a
+position where we need to connect the data that's in those different tools we
+use.
+
+If we are administering surveys using ODK, we might need to send metadata to a
+management system like Salesforce where we can analyze it, report on it and
+share it with other members of our team—maybe tracking the progress of a large
+study.
+
+In connecting these different tools we set up different data integration
```

Comment:
@jedbgold , i got here with my edits... would you move from this point forward checking each sentence for content-value? words cost $100, sentences cost $1000.